plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -420,14 +424,14 @@ private Slice writeDeletionVector(
 
         try {
             DataFileInfo newFileInfo = new DataFileInfo(
-                    sourceRelativePath,
+                    sourceReferencePath,
```

Comment:
Is this cloned path used to write new deletion vector files?

Reply:
No, the deletion vector path is under the current table location, see the `DeletionVectors#writeDeletionVectors`.
The sourceReferencePath is to build the `RemoveFileEntry` later in `DeltaLakeMetadata#commitMergeOperation`